packages/cli/src/daemon/log-requests.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express';
+import { LoggerProvider } from '@ceramicnetwork/common';
+import morgan from 'morgan';
+
+const ACCESS_LOG_FMT = 'ip=:remote-addr ts=:date[iso] method=:method original_url=:original-url base_url=:base-url path=:path http_version=:http-version req_header:req[header] status=:status content_length=:res[content-length] content_type=":res[content-type]" ref=:referrer user_agent=:user-agent elapsed_ms=:total-time[3] error_message=:error-message error_code=:error-code';
+
+export function logRequests(loggerProvider: LoggerProvider): any[] {
```

Comment:
I think this is a misnomer. This function is just returning middleware, not actually logging anything.

I think we should call it `buildHttpLog` (or something similar) and it can return the middleware as any object instead of an array.

Reply:
It returns a middleware, that _logs requests_ :) One discovers existence of this thing I guess by reading `app.use` clause first.

How do you feel about `app.use(buildHttpLog(loggerProvider))` vs `app.use(logRequests(loggerProvider))`?

Reply:
I guess it should be `app.use(loggerMiddleware(...))`

Reply:
It is always middleware put to `app.use` :) I was following the lead of a canonical list https://expressjs.com/en/resources/middleware.html There is no one middleware there, btw. Names are either idiomatic nouns (Morgan, session, timeout, slash), or agent nouns (body-parser, cookie-parser, errorHandler) or plain verbs (serve-static, expiry, uncapitalize).

`requestLogger` as the middle ground? It is a petty battle anyway.

Reply:
Like, after your next proposition, I give up and accept anything.

Reply:
Yeah I think you are right actually because it just takes a function. Up to you!